src/Philter.Aura/Philter.Aura.Data/Services/IMessagingService.cs

```diff
@@ -6,6 +9,8 @@ namespace Philter.Aura.Data.Services;
 [Coalesce, Service]
 public interface IMessagingService
 {
-	[Execute(SecurityPermissionLevels.AllowAuthorized)]
-	public MessageResource SendText(PhoneNumber to, string messagingServiceId, string message);
+    [Execute(SecurityPermissionLevels.AllowAuthorized)]
+    public Task<ItemResult<MessageResource>> SendText(PhoneNumber to, string messagingServiceId, string message);
+    [Execute(SecurityPermissionLevels.AllowAuthorized)]
```

Comment:
```suggestion
   
     [Execute(SecurityPermissionLevels.AllowAuthorized)]
```
nit, add 1 line of whitespace between method signatures